### PR TITLE
Add duration-select directive.

### PR DESCRIPTION
**Pull request by maxpeterson**

### CODE REVIEW

File: src/templates/type/duration-select.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<div class="duration-select-wrapper">
+    <table>
+        <tbody>
```
Comment (wytrych) [inline src/templates/type/duration-select.html]: Do you need the `tbody` ?

Comment (maxpeterson) [inline src/templates/type/duration-select.html]: I thought it was "the done thing".

File: Gruntfile.js [hunk 2]
```diff
@@ -108,7 +110,8 @@ module.exports = function (grunt) {
                     '<%= config.lib %>/angular-bind-html-compile/angular-bind-html-compile.js',
                     '<%= config.lib %>/angular-gettext/dist/angular-gettext.js',
 
-                    'dist/**/*.js',
+                    'src/**/*.js',
```
Comment (maxpeterson) [inline Gruntfile.js]: I changed this to make debugging easier during testing (otherwise it is uglifyed)

**Comment by maxpeterson** [comment]
@wytrych please review.

**Comment by wytrych** [comment]
Looks good !

**Pull request merged**